src/blockchain/watcher.js

```diff
@@ -571,7 +576,8 @@ const watcher = app => {
       subscribeApps();
       subscribeCappedMilestones();
       subscribeVault();
-
+      initNewEventQueue(app);
```

Comment:
I've moved initializing queue processing in here because after restarting, the queue will process remaining events in queue, so if we don't initialize it in here, the events will not process until a new event comes to queue and initialize queue processing, I hope my description be clear